admin/js/boldgrid-backup-admin-settings-autoupdate.js

```diff
@@ -66,51 +89,68 @@ BOLDGRID.SETTINGS = BOLDGRID.SETTINGS || {};
 				$pluginToggles = $bgBox.find( '.plugin-toggle' ),
 				$themeToggles = $bgBox.find( '.theme-toggle' ),
 				$pluginsDefault = $bgBox.find( '#toggle-default-plugins' ),
-				$themesDefault = $bgBox.find( '#toggle-default-themes' );
+				$themesDefault = $bgBox.find( '#toggle-default-themes' ),
+				$timelyUpdatesEnabled = $bgBox.find( '#timely-updates-enabled' ),
+				$timelyUpdatesDisabled = $bgBox.find( '#timely-updates-disabled' ),
+				$timelyUpdatesDays = $bgBox.find( '#timely-updates-days' );
 
 			// If the updates section is not in use, then just return.
-			if ( ! $pluginsDefault.data( 'toggles' ) ) {
-				return;
-			}
+			// if ( ! $pluginsDefault.data( 'toggles' ) ) {
+			// 	return;
+			// }
 
 			$wpcoreToggles.each( function() {
 				var $this = $( this );
 
-				$this
+				var $thisInput = $this
 					.next( 'input' )
-					.attr( 'name', 'autoupdate[wpcore][' + $this.data( 'wpcore' ) + ']' )
+					.attr( 'name', 'auto_update[wpcore][' + $this.data( 'wpcore' ) + ']' )
 					.val( $this.data( 'toggles' ).active ? 1 : 0 );
+				$this.change( function() {
+					if ( true == $this.prop( 'checked' ) ) {
```

Comment:
This code is being duplicated. Can we change that?

Reply:
created another function _onInputChange to handle this